dev/com.ibm.websphere.appserver.features/visibility/public/jaxb-2.2/com.ibm.websphere.appserver.jaxb-2.2.feature

```diff
@@ -17,12 +17,8 @@ IBM-Process-Types: client, \
  server
 Subsystem-Name: Java XML Bindings 2.2
 -features=\
-  com.ibm.websphere.appserver.classloading-1.0
--bundles=\
-  com.ibm.websphere.javaee.activation.1.1; require-java:="9"; location:="dev/api/spec/,lib/"; mavenCoordinates="javax.activation:activation:1.1", \
-  com.ibm.websphere.javaee.jaxb.2.2; location:="dev/api/spec/,lib/"; mavenCoordinates="javax.xml.bind:jaxb-api:2.2.12", \
-  com.ibm.ws.org.apache.geronimo.osgi.registry.1.1, \
-  com.ibm.ws.jaxb.tools.2.2.10
+  com.ibm.websphere.appserver.classloading-1.0, \
+  com.ibm.websphere.appserver.internal.optional.jaxb-2.2; ibm.tolerates:=2.3
```

Comment:
I think this line should be ```com.ibm.websphere.appserver.optional.jaxb-2.2; ibm.tolerates:=2.3``` so that we have a linear dependency chain.... but it might be ok as is.

Reply:
I totally agree that it's a better practice. Applied.